config/src/main/java/org/axonframework/config/AggregateConfigurer.java

```diff
@@ -386,4 +392,28 @@ public void shutdown() {
     public Class<A> aggregateType() {
         return aggregate;
     }
+
+    /**
+     * Registers subtypes of this aggregate to support aggregate polymorphism. Command Handlers defined on this
+     * subtypes will be considered part of this aggregate's handlers.
+     *
+     * @param subtypes subtypes in this polymorphic hierarchy
+     * @return this configurer for fluent interfacing
+     */
+    @SafeVarargs
+    public final AggregateConfigurer<A> withSubtype(Class<? extends A>... subtypes) {
+        return withSubtype(Arrays.asList(subtypes));
+    }
+
+    /**
+     * Registers subtypes of this aggregate to support aggregate polymorphism. Command Handlers defined on this
+     * subtypes will be considered part of this aggregate's handlers.
+     *
+     * @param subtypes subtypes in this polymorphic hierarchy
+     * @return this configurer for fluent interfacing
+     */
+    public AggregateConfigurer<A> withSubtype(Collection<Class<? extends A>> subtypes) {
```

Comment:
It is somewhat nitty-gritty, but shouldn't this method be called `withSubtypes` (plural)?